libs/designer-ui/src/lib/card/cardfooter.tsx

```diff
@@ -54,6 +67,14 @@ export const CardFooter: React.FC<CardFooterProps> = ({ commentBox, connectionDi
     defaultMessage: 'Comment',
     description: 'This is for a label for a badge, it is used for screen readers and not shown on the screen.',
   });
+  const SECURE_INPUTS_OUTPUTS_TITLE = intl.formatMessage({
```

Comment:
Hi @ecfan, could you please check these two texts?

Reply:
The ones that mention "secure inputs and outputs"? They look OK to me.

Reply:
Thanks!